unittests/unittest_udp_address.cpp

```diff
@@ -111,7 +109,9 @@ static void test_resolve_ipv4_connect_any ()
 
 static void test_resolve_ipv4_connect_anyport ()
 {
-    test_resolve ("127.0.0.1:*", NULL);
+    //  Not sure if that's useful but the TCP code allows it so why
```

Comment:
Hm. Does this have a defined behaviour on any platform? If not, it should be disallowed IMO.
The same applies to TCP.